src/preferences/PreferencesManager.js

```diff
@@ -528,7 +532,7 @@ define(function (require, exports, module) {
      * @return {boolean} true if a value was set
      */
     function setValueAndSave(id, value, options) {
-        DeprecationWarning.deprecationWarning("setValueAndSave called for " + id + ". Use set instead.");
```

Comment:
I think this is supposed to be called only once for all the extension's preferences anyway, but it probably wouldn't hurt to set that second parameter.